.github/workflows/scheduled-pod-level-pipeline.yml

```diff
@@ -4,80 +4,123 @@ on:
   schedule:
     - cron: "30 20 * * *" # Daily 02:30 AM in midnight
 
-defaults:
-  run:
-    working-directory: /go/src
-
 jobs:
-  ### Setup Litmus
-
-  Setup_Litmus_Infra:
-    container:
-      image: "litmuschaos/litmus-e2e:ci"
-      volumes:
-        - /home/ubuntu/.kube:/root/.kube
-        - /home/ubuntu/.aws:/root/.aws
-        - /etc/kubernetes:/etc/kubernetes
-      env:
-        KUBECONFIG: /root/.kube/config
-
-    runs-on: [self-hosted, pod-level]
 
+  Network_Chaos_Test:
+    if: always()
+    runs-on: ubuntu-latest
+    env:
+      CHAOS_NAMESPACE: "litmus"
+      APP_NS: "litmus"
+      UPDATE_WEBSITE: "true"
+      KUBECONFIG: /etc/rancher/k3s/k3s.yaml   
+      POD_LEVEL: "true"      
+    
     steps:
+    
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.16'
+
+      - uses: actions/checkout@v2
+        with:
+          repository: '${{ github.event.inputs.e2eTestRepo }}'
```

Comment:
These both inputs `e2eTestRepo` & `e2eTestBranch` won't be available in scheduled pipelines